styles/global.css

```diff
@@ -1,12 +1,32 @@
-body {
-  margin: 0;
-  background: linear-gradient(
+:root {
+  --background-color: linear-gradient(
     to right,
     rgb(3, 5, 58),
     rgb(3, 1, 29),
     rgb(17, 3, 56)
   );
+  --explanation: #534fc6;
+  --zoombutton: #534fc6;
+  --zoombutton-hover: #5955c6;
```

Comment:
Ik denk dat het beter is om algemene kleurnamen te geven ipv op basis van de specifieke context waar ze gebruikt worden. Vb: `zoombutton` kan `blue` ofzo worden dat dan bijvoorbeeld ook herbruikt kan worden voor de blauwe kleur van de toggle. Analoog met `explanation` etc...

Clean gedaan met die variabelen though 👏

Reply:
Hoop dat dit wat beter is?